jobs/delorean/suites/integration/rc/branch.yaml

```diff
@@ -0,0 +1,36 @@
+---
+- job:
+    name: delorean-suites/integration/rc/branch
+    display-name: 'integration-rc-branch'
+    project-type: pipeline
+    concurrent: false
+    parameters:
+      - string:
+          name: 'installationGitUrl'
+          default: 'git@github.com:integr8ly/installation.git'
+          description: '[REQUIRED] The installation repo'
+      - string:
+          name: 'suiteBranch'
+          default: 'integration-rc'
+          description: '[REQUIRED] The installation git branch to push new version changes'
+      - string:
+          name: 'productBranches'
+          default: '3scale-rc, amq-online-rc, fuse-online-rc, fuse-rc'
```

Comment:
Do we also include `fuse-rc` here? I don't think we currently have a branch job for this or an issue to add this branch.